UnitTest/DataStructuresTests/PrimeHashingFamilyTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿using DataStructures.Hashing;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace UnitTest.DataStructuresTests
+{
+    
+    public static class PrimeHashingFamilyTest
+    {
+        [Fact]
+        public static void CheckHashIdenticalInt()
+        {
+            PrimeHashingFamily hash1 = new PrimeHashingFamily(1);
+            PrimeHashingFamily hash2 = new PrimeHashingFamily(1);
+            int hashValue = "főzelék".GetHashCode();
+
+            for (int i = 1; i < hash1.NumberOfFunctions; i++)
+            {
+                hash1 = new PrimeHashingFamily(i);
```

Comment:
This is changing the condition of the for loop inside the loop, which is a bad practice.